src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
```

Comment:
We should also document:
- Quotas at the namespace level
- Metrics tab for each resource in a namespace
- Support for [priorityClassName](https://www.okteto.com/docs/1.25/reference/okteto-manifest/#priorityclassname-string-optional) and [accessMode](https://www.okteto.com/docs/1.25/reference/okteto-manifest/#persistentvolume-object-optional) for volumes created by `okteto up`